controllers/admin/AdminSuppliersController.php

```diff
@@ -231,6 +231,14 @@ public function renderForm()
                         'name' => 'name',
                     ),
                 ),
+                array(
+                    'type' => 'text',
+                    'label' => $this->trans('DNI', array(), 'Admin.Global'),
```

Comment:
Hey, are the NIF and NIE mandatory as well? I can see that there was a string _DNI / NIF / NIE_ in 1.6.

Reply:
No, it's just the DNI